docs/install/install-serving-with-yaml.md

```diff
@@ -34,6 +34,15 @@ To install the serving component:
 
 ## Install a networking layer
 
+The networking layer in Knative Serving routes incoming and outgoing traffic for your Knative installation.
+Knative enables you to choose a networking layer and then converts Knative’s resources
+into the specific configuration for your chosen networking solution.
+
+You can choose a networking layer based on what you are familiar with, what is
+already installed on your cluster, or what is suitable for your use case.
+For prototyping purposes, if you have no preference for a networking layer, choose Kourier.
+For production use cases, Istio might be a better choice because it has more features.
```

Comment:
How about "Istio is a common choice for production use."

I think highlighting Kourier for prototyping use is super-useful. I'm not quite as sure about making a recommendation for production use, but I don't fele strongly to suggest a different approach here.